Add catalog-db process for fcs

jbrown-xentity opened the pull request:
Since we were collaborating on the gists, it seemed prudent to put these in a collaborative and long term space...

@FuhuXia to add inventory once on a pause/stable for easier collaboration.

jbrown-xentity committed:
Add catalog-db process for fcs

jbrown-xentity committed:
parameterize new db name

Add wait functionality for cf8

jbrown-xentity committed:
Report possible failure

nickumia-reisys commented on docs/fcs_migration/02-fcs-catalog-db-restore.sh:
```diff
@@ -0,0 +1,77 @@
+cf scale catalog -i 0
+cf run-task catalog -c "ckan db upgrade"
+cf scale catalog -i 2
+cf run-task catalog -c "ckan search-index rebuild"
```
```suggestion
cf run-task catalog -c "ckan search-index rebuild -i -o -e" --name search-index-rebuild -k 2G -m 2G
```

nickumia-reisys committed:
Update docs/fcs_migration/02-fcs-catalog-db-restore.sh

-i (ignore errors)
-o (only index missing points)
-e (commit in realtime; does not slow down from testing)

nickumia-reisys committed:
update: don't use dashboard because naming is different between prod and staging; make storage size for new db a parameter

nickumia-reisys committed:
Merge branch 'main' into feature/fcs-migration-notes
